cli-plugins/apps/debug-workload.hbs.md

```diff
@@ -67,6 +67,64 @@ There are known errors that will make the workload enter in an error or unknown
 			- *Resolution*: Checkout this page on how to setup Git Authentication for Private repositories [Link to [Git authentication](https://docs.vmware.com/en/VMware-Tanzu-Application-Platform/1.1/tap/GUID-scc-git-auth.html)]
 
 - *TemplateRejectedByAPIServer*
-	- *Message*: Unable to apply object [ns/workload-name] for resource [source-provider] in supply chain [source-to-url]: failed to get unstructured [ns/workload-name] from api server: imagerepositories.source.apps.tanzu.vmware.com "workload-name" is forbidden: User "system:serviceaccount:ns:default" cannot get resource "imagerepositories" in API group "source.apps.tanzu.vmware.com" in the namespace "ns"
+	- *Message*: Unable to apply object `[ns/workload-name]` for resource `[source-provider]` in supply chain `[source-to-url]`: failed to get unstructured `[ns/workload-name]` from api server: imagerepositories.source.apps.tanzu.vmware.com "workload-name" is forbidden: User "system:serviceaccount:ns:default" cannot get resource "imagerepositories" in API group "source.apps.tanzu.vmware.com" in the namespace "ns"
 	- *Cause*: This error happens when the service account in the workload object does not have permissions to create objects that are stamped out by the supply chain.
 		- *Resolution*: This can be fixed by setting up the [Set up developer namespaces to use installed packages](https://docs.vmware.com/en/VMware-Tanzu-Application-Platform/1.3/tap/GUID-set-up-namespaces.html) with the required service account and permissions.
```

Comment:
- *Resolution*: This can be fixed by setting up the [Set up developer namespaces to use installed packages](THIS LINK SHOULD BE A RELATIVE LINK TO THE DOCS-TAP .hbs.md file/page, NOT A FULLY QUALIFIED URL TO THE PUBLIC DOCS) with the required service account and permissions. (sorry for the all caps, just wanted it to stand out - I'm not angry at all :) )